### PR TITLE
Add reset_planning_scene feature for ROS

### LINKED ISSUE #107: Allow to reset planning scene
There should be a method to reset the planning scene completely when using ROS backend.

### PR DESCRIPTION
**Pull request by beverlylytle**
Closes #107.

### What type of change is this?

- [ ] Bug fix in a **backwards-compatible** manner.
- [x] New feature in a **backwards-compatible** manner.
- [ ] Breaking change: bug fix or new feature that involve incompatible API changes.
- [ ] Other (e.g. doc update, configuration, etc)

### Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I added a line to the `CHANGELOG.rst` file in the `Unreleased` section under the most fitting heading (e.g. `Added`, `Changed`, `Removed`).
- [x] I ran all tests on my computer and it's all green (i.e. `invoke test`).
- [x] I ran lint on my computer and there are no errors (i.e. `invoke lint`).
- [x] I added new functions/classes and made them available on a second-level import, e.g. `compas_fab.robots.Configuration`.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by beverlylytle** [comment]
And since I was there, I removed all references to ABCMeta.

**Comment by beverlylytle** [comment]
Ups, that cone_tool.json just slipped in there... I thought I just updated that....

File: src/compas_fab/backends/interfaces/backend_features.py [hunk 1]
```diff
@@ -197,6 +184,34 @@ def get_planning_scene(self, options=None):
...
+
+        Parameters
+        ----------
+        options : dict, optional
+            Dictionary containing kwargs for arguments specific to
+            the client being queried.
+
+        Returns
+        -------
+        :class:`compas_fab.robots.planning_scene.PlanningScene`
```
Comment (beverlylytle) [inline src/compas_fab/backends/interfaces/backend_features.py:210]:
```suggestion
        ``None``
```

**Review by gonzalocasas — APPROVED** [review]
LGTM! 

Joey nice

**Pull request merged**